proposals/stage-3/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -13,12 +13,20 @@ A rough description of the project in less than 100 words.
 Please refer to [the Cross Project Council's Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md).
 Please keep your answer to less than 250 words.
 
+## Impact and consumers of the project
```

Comment:
This does not need to be modified (and arguably shouldn't). It is a historic document

Reply:
oops, I thought it is duplicated. Will revert this.